Makefile

```diff
@@ -133,13 +124,13 @@ test-list: test/test-list.js
 test/test-list.js: force
 	@git ls-files -o -c test/feature | node build/build-test-list.js > $@
 
-test-interpret: test/unit/runtime/test_interpret.js
+test-interpret: test/unit/runtime/traceur-interpreter.js
 	./traceur $^
 
-test-interpret-throw: test/unit/runtime/throwsError.js
+test-interpret-throw: test/unit/runtime/resources/throwsError.js
```

Comment:
OK.

 The way I think of this is that `resources/**` are files used by other tests and should not be picked up by any test runner. I think at one point the mocha test had a glob that excluded resources/**.